embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/logs/EmbraceLogRecordData.kt

```diff
@@ -0,0 +1,43 @@
+package io.embrace.android.embracesdk.internal.logs
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import io.opentelemetry.sdk.logs.data.LogRecordData
+
+/**
+ * Serializable representation of [EmbraceLogRecordData]
+ */
+@JsonClass(generateAdapter = true)
+internal data class EmbraceLogRecordData(
+    @Json(name = "trace_id")
+    val traceId: String,
+
+    @Json(name = "span_id")
+    val spanId: String,
+
+    @Json(name = "time_unix_nano")
+    val timeUnixNanos: Long,
+
+    @Json(name = "severity_number")
+    val severityNumber: Int,
+
+    @Json(name = "severity_text")
+    val severityText: String?,
+
+    @Json(name = "body")
+    val body: EmbraceBody,
+
+    @Json(name = "attributes")
+    val attributes: Map<String, String> = emptyMap()
```

Comment:
This should actually be a List of Pair<String, Any>. The whole list thing is an OTLP decision for performance reasons, and we are moving to allowing types other than Strings as attribute values. The v2 endpoints will support this, so this is something the `EmbraceSpanData` will be be changed to do that as well

Reply:
Changed to `List<Pair<String, Any>>`.

Reply:
I've rolled back the change from `Map` to `List<Pair>` because we can not use kotlin.Pair with a generated Moshi class adapter; I think changing that is outside the scope of this PR and part of the work Fran is doing.